homeassistant/components/lookin/config_flow.py

```diff
@@ -92,10 +95,6 @@ async def async_step_discovery_confirm(
         """Confirm the discover flow."""
         assert self._host is not None
         if user_input is None:
-            self.context["title_placeholders"] = {
-                "name": self._name,
-                "host": self._host,
-            }
             return self.async_show_form(
                 step_id="discovery_confirm",
                 description_placeholders={"name": self._name, "host": self._host},
```

Comment:
```suggestion
                description_placeholders={"name": self._name or "LOOKin", "host": self._host},
```
Would it make sense to apply here also on the `description_placeholders`?
It would be needed for https://github.com/home-assistant/core/pull/127103

Reply:
I'm not sure, this PR is about making the title_placeholders items [str, str]

Reply:
If you prefer... I will adjust it in a separate PR then.